### PR TITLE
Balance: synthskin arms no longer malf below a 10 combined damage threshold

### PR DESCRIPTION
**Pull request by JohnDoe50974**
## About The Pull Request
Essentially a rehearse of this Gitlab merge request, but with a more sensible threshold value, and applies to synthetics only: https://gitlab.com/cmdevs/colonial-warfare/-/merge_requests/2102
If the combined brute/burn damage on a synthskin arm or hand is at or below 10 points, the malfunction proc will not trigger.

## Why It's Good For The Game
Prevents situations where punching a synth once or throwing a piece of paper at one's arm causes their arms to malf and drop held items until smartpack charge/welder/cable coil was expended to repair it, due to synths lacking the natural hp regeneration that human mobs have, which is an unreasonably expensive process both time and resources-wise for something so comparatively meaningless. A marine wearing light armor receives infinitesimal damage from another human's punch, and said damage amount heals itself almost immediately, which leads to comical situations where synths appear more fragile than your average human. More importantly, this balance change does not affect regular prosthetic limbs' balance, which was what Nanu requested to be done in the previous MR after closing it.

In short, it will make synthetic gameplay less frustrating and greatly reduce fussing over, say 0.534568 points of brute damage causing a synth to drop what they were holding, which feels completely nonsensical and unrealistic.

## Changelog

:cl: 50RemAndCounting
balance: synthskin arms and hands will no longer malfunction and drop items if the combined brute and burn damage on them does not exceed 10 points.
/:cl:

### CODE REVIEW

File: code/modules/organs/limbs.dm [hunk 1]
```diff
@@ -1131,7 +1131,12 @@ treat_grafted var tells it to apply to grafted but unsalved wounds, for burn kit
 	return ((status & LIMB_BROKEN) && !(status & LIMB_SPLINTED))
 
 /obj/limb/proc/is_malfunctioning()
-	return ((status & (LIMB_ROBOT|LIMB_SYNTHSKIN)) && prob(brute_dam + burn_dam))
+{
+	if(status & LIMB_ROBOT)
+		return ((status & LIMB_ROBOT) && prob(brute_dam + burn_dam))
+	else if(status & LIMB_SYNTHSKIN && (brute_dam + burn_dam) > 10)
+		return (status & LIMB_SYNTHSKIN) && (prob(brute_dam + burn_dam))
+}
```
Comment (stanalbatross) [inline code/modules/organs/limbs.dm:1139]:
```suggestion
	if(status & LIMB_ROBOT)
		return ((status & LIMB_ROBOT) && prob(brute_dam + burn_dam))
	else if(status & LIMB_SYNTHSKIN && (brute_dam + burn_dam) > 10)
		return (status & LIMB_SYNTHSKIN) && (prob(brute_dam + burn_dam))
```
also I don't think you need this **((status & LIMB_ROBOT)**

**Review by stanalbatross — CHANGES_REQUESTED** [review]

File: code/modules/organs/limbs.dm (hunk 1 above)
Comment (JohnDoe50974) [reply code/modules/organs/limbs.dm]: How is the game going to know to trigger the malf proc on LIMB_ROBOT limbs? I'd rather not touch it as that part was the original proc.

Comment (stanalbatross) [reply code/modules/organs/limbs.dm]: kk understandable

File: code/modules/organs/limbs.dm [hunk 2]
```diff
@@ -1131,7 +1131,10 @@ treat_grafted var tells it to apply to grafted but unsalved wounds, for burn kit
 	return ((status & LIMB_BROKEN) && !(status & LIMB_SPLINTED))
 
 /obj/limb/proc/is_malfunctioning()
-	return ((status & (LIMB_ROBOT|LIMB_SYNTHSKIN)) && prob(brute_dam + burn_dam))
+	if(status & LIMB_ROBOT)
+		return ((status & LIMB_ROBOT) && prob(brute_dam + burn_dam))
+	else if(status & LIMB_SYNTHSKIN && (brute_dam + burn_dam) > 10)
+		return (status & LIMB_SYNTHSKIN) && (prob(brute_dam + burn_dam))
```
Comment (stanalbatross) [inline code/modules/organs/limbs.dm:1137]:
```suggestion
	if(status & LIMB_ROBOT)
		return prob(brute_dam + burn_dam)
	else if(status & LIMB_SYNTHSKIN && (brute_dam + burn_dam) > 10)
		return prob(brute_dam + burn_dam)
```

**Review by stanalbatross — CHANGES_REQUESTED** [review]

**Review by stanalbatross — APPROVED** [review]
